### DISCUSSION TITLE
Configured shortcut doesn't match display in Kit app.

### DISCUSSION THREAD

**Discussion by nicksergeant** [johnlindquist/kit#634] [Errors]
I've configured a shortcut here for `command option r`, but the Kit app displays `cmd+ctrl+opt+r`. Strangely, neither of these shortcuts works, either.

![image](https://user-images.githubusercontent.com/27332/144654242-92720222-99cb-4629-a6f9-8586849a70fc.png)

**Comment by johnlindquist** [comment]
Thanks, investigating...

**Reply by johnlindquist** [reply]
1. Pressing the shortcut:  Both of those shortcuts work for me, hmm 🤔. Is that script in the main kenv?

2. Shortcut display: I do see it displaying an incorrect shortcut. That's some Mac/Windows shortcut parsing I'll get fixed for the next build.

**Reply by nicksergeant** [reply]
Huh, I went back to test this again and it is working for me now. I wonder if I was thrown off by the KB shortcut displayed in the app and just mentally attempting to use the incorrect one each time...

At any rate, I think all good once the display issue is fixed.

Thank you!
